debuggers/gdb/gdbdebugger.js

```diff
@@ -60,8 +58,20 @@ define(function(require, exports, module) {
                     ["autoshow", "true"]
                 ]);
             });
-
+            
+            // must register ASAP, or debugger won't be ready for reconnects
             debug.registerDebugger(TYPE, plugin);
+
+            // writeFile root is workspace directory, unless given ~
+            var shimPath = "~/bin/c9gdbshim.js";
```

Comment:
MD5 or other versioning would be fine, so long as we have a symlink or some other mechanism that always points to the latest revision from a known path, e.g., `~/.c9/bin/c9gdbshim.js` is a link for `~/.c9/bin/c9gdbhim-<md5>.js`?